java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -127,6 +127,7 @@ disabled.modules=\
     org.netbeans.modules.debugger.jpda.js,\
     org.netbeans.modules.debugger.jpda.jsui,\
     org.netbeans.modules.debugger.jpda.kit,\
+    org.netbeans.modules.debugger.jpda.trufflenode,\
```

Comment:
I am not really sure about this one. Will https://github.com/graalvm/graal-js-archetype continue to work/be debugable?

Reply:
@entlicher should know...

Reply:
The `jpda.trufflenode` binds to the node.js support in NetBeans. As it does not make sense to use an old node.js support from NetBeans in VSCode, it's fine to remove it.
We would need to invent a different approach in VSCode how to execute debugger for node.js projects via Truffle on GraalVM to connect node.js & Java debugging.